src/transforms/lua/v1/mod.rs

```diff
@@ -223,30 +225,31 @@ impl mlua::UserData for LuaEvent {
         methods.add_meta_method_mut(
             mlua::MetaMethod::NewIndex,
             |_lua, this, (key, value): (String, Option<mlua::Value<'lua>>)| {
+                let key_path = parse_target_path(key.as_str()).map_err(|e| e.to_lua_err())?;
```

Comment:
Do we want to propagate an error here?

Reply:
That seems reasonable to me. We've been talking about deprecating the V1 lua transform for quite some time now, so this might not matter too much.